lottie/src/main/java/com/airbnb/lottie/Lottie.java

```diff
@@ -20,7 +20,6 @@ public static void initialize(@NonNull final LottieConfig lottieConfig) {
     L.setCacheProvider(lottieConfig.cacheProvider);
     L.setTraceEnabled(lottieConfig.enableSystraceMarkers);
     L.setNetworkCacheEnabled(lottieConfig.enableNetworkCache);
-    L.setNetworkCacheEnabled(lottieConfig.enableNetworkCache);
```

Comment:
I assume this double-call was just a typo, but I can revert if it was intentional.